test/e2e/setup_test.go

```diff
@@ -39,9 +39,6 @@ var (
 )
 
 func TestMain(m *testing.M) {
-	if err := flag.Set("logtostderr", "true"); err != nil {
```

Comment:
In `klog` this is [true by default](https://github.com/kubernetes/klog/pull/39).